client/src/App.js

```diff
@@ -0,0 +1,116 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import axios from 'axios';
+import { BootstrapTable, TableHeaderColumn } from 'react-bootstrap-table';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    // this.state = {
+    //   name: "Ammar"
+    // }
+    this.state = {};
+    this.getData = this.getData.bind(this);
+    this.sendData = this.sendData.bind(this);
+    this.updateData = this.updateData.bind(this);
+    this.deleteData = this.deleteData.bind(this);
```

Comment:
There is a nice module that does this called [auto-bind-es5](https://www.npmjs.com/package/auto-bind-es5) :smile: It's shorter to write